### PR TITLE
Add support for OpenAPI v3.

### PR DESCRIPTION
**Pull request by bpolaszek**
This PR adds a parser for OpenAPI v3.

@dunglas Can you please review?

### CODE REVIEW

**Comment by bpolaszek** [comment]
Fix small typo in README.

File: README.md [hunk 1]
```diff
@@ -40,10 +40,17 @@ import parseSwaggerDocumentation from 'api-doc-parser/lib/swagger/parseSwaggerDo
 parseSwaggerDocumentation('https://demo.api-platform.com/docs.json').then(({api}) => console.log(api));
 ```
 
+**OpenApi v3**
```
Comment (dunglas) [inline README.md]: I think we can promote it over Swagger

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 [![npm version](https://badge.fury.io/js/%40api-platform%2Fapi-doc-parser.svg)](https://badge.fury.io/js/%40api-platform%2Fapi-doc-parser)
 [![MIT Licence](https://badges.frapsoft.com/os/mit/mit.svg?v=103)](https://opensource.org/licenses/mit-license.php)
 
-`api-doc-parser` is a JavaScript (ES6) library to parse [Hydra](http://hydra-cg.com) or Swagger API documentations and transform them
+`api-doc-parser` is a JavaScript (ES6) library to parse [Hydra](http://hydra-cg.com), Swagger and OpenAPI documentations and transform them
```
Comment (dunglas) [inline README.md]: Can you add a link to OpenAPI?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@api-platform/api-doc-parser",
-  "version": "0.7.2",
+  "version": "0.7.3",
```
Comment (dunglas) [inline package.json]: Can you revert this? It's the CI that takes care of this.

File: README.md (hunk 1 above)
Comment (bpolaszek) [reply README.md]: Do you mean we should remove everything related to Swagger in the Readme?

File: README.md (hunk 2 above)
Comment (bpolaszek) [reply README.md]: Sure.

File: package.json (hunk 3 above)
Comment (bpolaszek) [reply package.json]: Great :+1:

**Comment by soyuka** [comment]
thanks @bpolaszek !

**Pull request merged**